Gemfile.lock

```diff
@@ -105,6 +105,7 @@ GEM
 
 PLATFORMS
   arm64-darwin-21
+  arm64-darwin-23
```

Comment:
My system adds `arm64-darwin-22`. Could we include that in this file as well?
```suggestion
  arm64-darwin-22
  arm64-darwin-23
```

Reply:
@amyleadem you can add platforms via `bundle lock --add-platform  arm64-darwin-22`.

Reply:
Resolved in [8dae60c](https://github.com/uswds/uswds-site/pull/2577/commits/8dae60c9b06b6fb175694f10fdafb0f15fe8ceb7)